src/broker/controller.h

```diff
@@ -63,7 +62,7 @@ struct ControllerListener {
 
 struct Controller {
         Broker *broker;
-        BusSELinuxID *sid;
+        char *context;
```

Comment:
`context`? Can't we come up with something more descriptive? I'd really prefer secctx or seclabel. You already call it `seclabel` on the peer, so why not here?

Also, are you sure we can treat *all* seclabels as 0-terminated strings? So far we always tried to treat them as sized blob and just added a 0-termination to make them pseudo-printable. I am aware that most (all?) LSMs use strings there, but is that guaranteed?

Reply:
I'll rename to seclabel. I could then also store n_seclabel, though it is totally fine to rely on these things being null-terminated as our helpers guarantee this (and in particular libselinux requires this). Othe LMSs could in principle have seclablels that contain null-bytes, in which case the length would be needed if we ever made use of it, so I'll add it now for completeness.